monai/transforms/transform.py

```diff
@@ -77,7 +65,6 @@ def set_random_state(
         self.R = np.random.RandomState()
         return self
 
-    @abstractmethod
     def randomize(self, data: Any) -> None:
```

Comment:
shall we support None here and in the `RandomizableTransform`?  (if it doesn't break anything...)
```suggestion
    def randomize(self, data: Optional[Any] = None) -> None:
```

Reply:
I shouldn't have enabled automatic merging! I would quite like the default of `data` for like `Randomizable` and `RandomizableTransform` to default to `None`. Currently I have to use a lot of `super().randomize(None)`, but I'd rather have `super().randomize()`. However, this is incompatible with certain child classes. E.g., `RandFlip` takes `np.ndarray` as argument, which is incompatible with `None`.

The alternative is to switch all child classes from e.g., `np.ndarray` to `Optional[np.ndarray]`. What do you think?

Reply:
thanks! it's just a very minor issue, I think the current one is also good, I forgot the mypy errors...